test/apps/data-filter/src/app.jsx

```diff
@@ -5,25 +5,45 @@ import {createRoot} from 'react-dom/client';
 import DeckGL, {GeoJsonLayer, COORDINATE_SYSTEM} from 'deck.gl';
 import {DataFilterExtension} from '@deck.gl/extensions';
 
-import DATA from './data-sample';
+import {DATA, COLORS, SHAPE_NAMES, SIZES} from './data-sample';
 
 const dataFilterExtension = new DataFilterExtension({
+  categorySize: 4,
   filterSize: 2,
   softMargin: true,
   countItems: true
 });
 
-const INITIAL_VIEW_STATE = {
-  longitude: -122.45,
-  latitude: 37.78,
-  zoom: 11
+const INITIAL_VIEW_STATE = {longitude: -122.45, latitude: 37.78, zoom: 12};
+
+const CONTROLS = {animate: false};
+const LABELS = {};
```

Comment:
Would be nice to convert apps to typescript as we modify them, and type things in the code. Helps readability and helps check types.